### PR TITLE
Add API test for "Product not found" scenario in Product API

### PR DESCRIPTION
**Pull request by uLucasFraga**
### Description 

Add API tests for `Product not found` scenarios in the task can be [seen here](https://github.com/stefanteixeira/demo-playwright-test/issues/32).

### How to test

- `yarn install && yarn api:start && yarn test:api`

### Checklist

- [x] Branch name follows the pattern: `concise-feature-description`
- [x] PR has a descriptive name
- [x] PR branch is up-to-date with `main` (if not, rebase it)
- [x] Double-check the quality of [commit messages](http://chris.beams.io/posts/git-commit/)

### CODE REVIEW

File: tests/api/product.api.test.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const { test, expect } = require('@playwright/test')
 const { StatusCodes } = require('http-status-codes')
-const { POST_SUCESS, DELETE_SUCESS, PUT_SUCESS, TOKEN_INVALID, NOME_JA_USADO } = require('serverest/src/utils/constants')
-const { getAuthToken, getProductBody, getProductId } = require('../../lib/helpers')
+const { POST_SUCESS, DELETE_SUCESS, PUT_SUCESS, TOKEN_INVALID, NOME_JA_USADO, IDPRODUTO_INVALIDO } = require('serverest/src/utils/constants')
```
Comment (stefanteixeira) [inline tests/api/product.api.test.js:3]:
```suggestion
const {
  POST_SUCESS,
  DELETE_SUCESS,
  PUT_SUCESS,
  TOKEN_INVALID,
  NOME_JA_USADO,
  IDPRODUTO_INVALIDO
} = require('serverest/src/utils/constants')
```

File: tests/api/product.api.test.js [hunk 2]
```diff
@@ -16,6 +16,19 @@ test.describe.parallel('Product API', () => {
     expect(response.status()).toEqual(StatusCodes.OK)
   })
 
+  test('retrieves a product non-existing via query params', async ({ request }) => {
```
Comment (stefanteixeira) [inline tests/api/product.api.test.js:19]:
```suggestion
  test('returns an empty list when a product does not exist', async ({ request }) => {
```

File: tests/api/product.api.test.js [hunk 3]
```diff
@@ -34,6 +47,16 @@ test.describe.parallel('Product API', () => {
     expect(response.status()).toEqual(StatusCodes.OK)
   })
 
+  test('retrieves a product non-existing via a path in the URL', async ({ request }) => {
```
Comment (stefanteixeira) [inline tests/api/product.api.test.js:50]:
```suggestion
  test('fails to retrieve a product when it does not exist', async ({ request }) => {
```

**Comment by stefanteixeira** [comment]
Great work @uLucasFraga 👏 
Added small suggestions above, thanks!

File: tests/api/product.api.test.js (hunk 1 above)
Comment (uLucasFraga) [reply tests/api/product.api.test.js:3]: oh, lint haha...ok!

File: tests/api/product.api.test.js (hunk 2 above)
Comment (uLucasFraga) [reply tests/api/product.api.test.js:19]: changed!

**Review by stefanteixeira — APPROVED** [review]
🚀

**Pull request merged**